### ISSUE TITLE
ENH: Speed up Usher webversion

### ISSUE THREAD

**Issue by corneliusroemer** [yatisht/usher#263]
Usher is such an amazing tool. Unfortunately, the run time of analyses using the phyloplace website are quite long. So long that I don't use Usher as much as I should.

To place say 50 sequences with 1000 context it does run for ~5min or more.

Does the number of context samples make it run much longer? Are there ways I can get it to be faster?

I'm curious what is the bottleneck for the analyses. I should probably try to run Usher locally to see whether this is faster and feasible for my use case.

If tree size is a problem: I'm only really interested in BA.2/4/5/.75 right now - have you considered making a version that ignores old stuff that's no longer relevant?

I do have the feeling that runtimes used to be quite a bit faster in the past.

**Comment by yatisht** [comment]
Thanks for the feedback, @corneliusroemer.

@AngieHinrichs are we using `usher-sampled-server` for the web version now? That should significantly speed up the tool by preloading the MAT and placing multiple samples in parallel.

**Comment by AngieHinrichs** [comment]
Since both fasta sequence placement and name lookup have been sped up, can we close this now?  😄

**Comment by russcd** [comment]
Yup. This is resolved.
